recipes/leftbar/callbox/callbox.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <dt-item-layout class="dt-recipe-callbox">
+    <template #default>
+      <div class="dt-recipe-callbox--video">
+        <!-- @slot Slot for video stream -->
+        <slot name="video" />
+      </div>
+    </template>
+    <template #bottom>
+      <dt-item-layout :class="['dt-recipe-callbox--main-content', borderClass]">
+        <template #default>
+          <dt-item-layout class="dt-recipe-callbox--top-content">
+            <template #left>
+              <dt-avatar
+                v-if="shouldShowAvatar"
+                :image-src="avatarSrc"
+                :full-name="avatarFullName"
+                size="sm"
+              />
+            </template>
+            <template #default>
+              <div class="dt-recipe-callbox--title">
+                {{ title }}
+              </div>
+            </template>
+            <template #subtitle>
+              <dt-item-layout class="dt-recipe-callbox--subtitle">
+                <template #default>
+                  <!-- @slot Slot for call center badge -->
+                  <slot name="badge">
+                    <dt-badge
+                      v-if="badgeText"
+                      :class="badgeClass"
+                      :text="badgeText"
+                    />
+                  </slot>
+                </template>
+                <template #bottom>
+                  <!-- @slot Slot for subtitle -->
+                  <slot name="subtitle" />
+                </template>
+              </dt-item-layout>
+            </template>
+            <template #right>
+              <div class="dt-recipe-callbox--right">
+                <!-- @slot Slot for right icons -->
+                <slot name="right" />
+              </div>
+            </template>
+          </dt-item-layout>
+        </template>
+        <template #bottom>
+          <div class="dt-recipe-callbox--bottom-content">
+            <slot name="bottom" />
+          </div>
+        </template>
+      </dt-item-layout>
+    </template>
+  </dt-item-layout>
+</template>
+
+<script>
+import { BADGE_COLORS, BORDER_COLORS } from './callbox_constants';
+import DtItemLayout from '@/components/item_layout/item_layout.vue';
+import DtAvatar from '@/components/avatar/avatar.vue';
+import DtBadge from '@/components/badge/badge.vue';
+
+export default {
+  name: 'DtRecipeCallbox',
+
+  components: { DtBadge, DtAvatar, DtItemLayout },
+
+  props: {
+    /**
+     * Text for the badge element
+     */
+    badgeText: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * Color for the badge element
+     * @values null, warning
+     */
+    badgeColor: {
+      type: String,
+      default: null,
+      validator: (color) => !!color || Object.keys(BADGE_COLORS).includes(color),
+    },
+
+    /**
+     * Optional avatar image url.
+     * If not provided it will use the initial of the name.
+     */
+    avatarSrc: {
+      type: String,
+      default: '',
+    },
```

Comment:
probably need to also pass seed or initials background color will constantly change.

Reply:
Added